galite-tests/src/test/kotlin/org/kopi/galite/tests/dsl/ReportDSLTests.kt

```diff
@@ -70,14 +70,36 @@ class ReportDSLTests: VApplicationTestBase() {
     assertEquals(listOf(20, 20, 30), report.getRowsForField(report.ageClt))
   }
 
- /* @Test
-  fun `test report triggers`() {
+  @Test
+  fun `test report rows`() {
     val report = ReportWithData()
     val reportModel = report.model
 
-    assertEquals(true, reportModel.hasTrigger(VConstants.TRG_INIT))
-    assertEquals(true, reportModel.hasTrigger(VConstants.TRG_PREBLK))
-  }*/
+    assertEquals(3, report.reportRows.size)
+
```

Comment:
change your test to : 
```
    val report = ReportWithData()
    report.model.initReport()
    val reportModel = report.model.model

    assertEquals(4, reportModel.getVisibleRowCount())
```
Rq: there is an default row in the report called separator